addons/website_sale/views/sale_order_views.xml

```diff
@@ -53,6 +76,26 @@
         <field name="help">You don't have any order from the website.</field>
     </record>
 
+    <!-- Server action to send multiple recovery email-->
+    <record id="send_recovery_email_action_server" model="ir.actions.server">
```

Comment:
I would rename the xml_id to something like ir_actions_server_sale_cart_recovery, for the same reasong as already mentioned before :) . I know, naming conventions ...